src/Plugin/RulesExpression/ActionSet.php

```diff
@@ -27,7 +27,17 @@ protected function allowsMetadataAssertions() {
    * {@inheritdoc}
    */
   public function executeWithState(ExecutionStateInterface $state) {
+    // Get hold of actions.
+    // @todo See if we can add getExpressions method of ExpressionContainerBase.
+    $actions = [];
```

Comment:
Actually I think its necessary because $this->actions is really an iterable object and not an array. This is part of the code coming from the included PR #452.